setup.py

```diff
@@ -24,14 +30,12 @@
         "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
         "Natural Language :: English",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
```

Comment:
Personally, I am not a fan of removing old version of python too quickly.

What is the rationale?

Reply:
These versions were removed because of importlib compatibility; however, we got rid of that and they should now be reinstated.